karate-tests/src/test/java/patients/TestParallel.java

```diff
@@ -16,7 +16,7 @@ void testDevParallel() {
                 .outputJunitXml(true)
                 .tags("~@sandbox-only", "~@smoke-only")
                 .karateEnv("veit07")
-                .parallel(3);
+                .parallel(2);
```

Comment:
Reduced parallel count to reduce the race condition of version mismatch errors on updates for p9 patients